editor/writing-flow/index.js

```diff
@@ -20,6 +26,8 @@ class WritingFlow extends Component {
 
 		this.onKeyDown = this.onKeyDown.bind( this );
 		this.bindContainer = this.bindContainer.bind( this );
+
+		this.verticalRect = null;
```

Comment:
Arriving at this pull request hopefully confused about what this variable is. Can we add some inline comments?

Reply:
Might be good to make an issue :)

Reply:
Comment coming.